client/src/providers/AuthContext.js

```diff
@@ -2,20 +2,26 @@ import React, { useReducer } from 'react';
 
 const AuthContext = React.createContext();
 
+const rolePriority = ['superuser', 'ministry_of_health', 'health_authority', 'employer'];
```

Comment:
NIT,  but maybe this could go in a constants file

Reply:
Ahh good nit you're right